src/context/WalletProvider/NativeWallet/components/LegacyTwoFactor.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Alert,
+  AlertIcon,
+  Button,
+  FormControl,
+  FormErrorMessage,
+  Input,
+  ModalBody,
+  ModalHeader,
+} from '@chakra-ui/react'
+import { Vault } from '@shapeshiftoss/hdwallet-native-vault'
+import { useState } from 'react'
+import { FieldValues, useForm } from 'react-hook-form'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router-dom'
+import { Text } from 'components/Text'
+
+// @TODO(NeOMakinG): Change this with the mnemonic of the legacy account
+const DUMMY_MNEMONIC = 'yolo yolo yolo yolo yolo yolo yolo yolo yolo yolo yolo yolo2'
+
+export const LegacyTwoFactor = () => {
+  const [error, setError] = useState<boolean | string>(false)
+  const history = useHistory()
+  const {
+    handleSubmit,
+    register,
+    formState: { errors, isSubmitting },
+  } = useForm({ shouldUnregister: true })
+
+  const translate = useTranslate()
+
+  const onSubmit = async (values: FieldValues) => {
+    try {
+      await new Promise((resolve, reject) => resolve(null))
+      // TODO: use response to create wallet vault.
+      const vault = await Vault.create(undefined, false)
+      vault.meta.set('createdAt', Date.now())
+      vault.set('#mnemonic', DUMMY_MNEMONIC)
+      history.push('/native/legacy/login/success', { vault })
+    } catch (err) {
+      setError(translate('walletProvider.shapeShift.legacy.invalidTwoFactor'))
+    }
+
+    return
+  }
```

Comment:
We need to avoid having two functions that create the Vault. Find a way to consolidate this.